test/e2e/makeGetCall.ts

```diff
@@ -1,40 +1,49 @@
-import { useQuery } from "@tanstack/react-query";
-import { Address, Cell, TonClient } from "ton";
-import BN from "bn.js";
+import {
+  Address,
+  Cell,
+  TonClient,
+  TupleBuilder,
+  TupleItemCell,
+  TupleReader,
+  Tuple,
+  TupleItemInt,
+} from "ton";
 
 function _prepareParams(params: any[] = []) {
-  return params.map((p) => {
+  const paramsTuple = new TupleBuilder();
+  params.forEach((p) => {
     if (p instanceof Cell) {
-      return ["tvm.Slice", p.toBoc({ idx: false }).toString("base64")];
-    } else if (p instanceof BN) {
-      return ["num", p.toString(10)];
+      paramsTuple.writeCell(p);
+    } else if (typeof p === "bigint") {
+      paramsTuple.writeNumber(p);
     }
-
     throw new Error("unknown type!");
   });
+  return paramsTuple.build();
 }
 
-type GetResponseValue = Cell | BN | null;
+type GetResponseValue = Cell | bigint | null;
 
-function _parseGetMethodCall(
-  stack: [["num" | "cell" | "list", any]]
-): GetResponseValue[] {
-  return stack.map(([type, val]) => {
-    switch (type) {
-      case "num":
-        return new BN(val.replace("0x", ""), "hex");
+function _parseGetMethodCall(stack: TupleReader): GetResponseValue[] {
+  const parsedItems: GetResponseValue[] = [];
+  while (stack.remaining) {
+    const item = stack.pop();
+    switch (item.type) {
+      case "int":
+        parsedItems.push((item as TupleItemInt).value);
       case "cell":
-        return Cell.fromBoc(Buffer.from(val.bytes, "base64"))[0];
-      case "list":
-        if (val.elements.length === 0) {
-          return null;
+        parsedItems.push((item as TupleItemCell).cell);
+      case "tuple":
+        if ((item as Tuple).items.length === 0) {
+          parsedItems.push(null);
         } else {
           throw new Error("list parsing not supported");
         }
       default:
-        throw new Error(`unknown type: ${type}, val: ${JSON.stringify(val)}`);
+        throw new Error(`unknown type: ${item.type}`);
     }
-  });
+  }
+  return parsedItems;
 }
 
 export async function makeGetCall<T>(
```

Comment:
is `makeGetCall` still used anywhere after adapting sandbox?

Reply:
There is this `sources-registry-only-set-code.fc` which is used in the tests. It has a `am_i_replaced` get method which is called in those test. Should we add another  wrapper for this dummy contract?

Reply:
Yes, just so we can lost `makeGetCall` altogether